src/common/zh.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
How come got two files for zh.json? I see `src/common/i18n/translations/zh.json` and `src/common/zh.json`